nlp_primitives/version.py

```diff
@@ -0,0 +1 @@
+__version__ = '2.5.0'
```

Comment:
Now that we have added this file, should we add a `test_version.py` file like we have in other repos?

Also, we need to update `release.md` to point to the proper places where the version needs to be changed: here and in `test_version.py` if we add that. Don't need to refer to `setup.py` any more.